core/nmf-composites/generic-composite-model/src/main/java/esa/mo/nmf/MonitorAndControlNMFAdapter.java

```diff
@@ -483,36 +483,43 @@ public Boolean onSetValue(ParameterRawValueList newRawValues)
   {
     boolean result = true;
     for (ParameterRawValue newRawValue : newRawValues) {
-
-      Object value;
-      if (isReadOnly(newRawValue.getParamInstId())) {
-        result = false;
-        continue;
-      }
-      Field param = parameterMapping.get(newRawValue.getParamInstId());
-
-      if (param.getType() == double.class) {
-        value = HelperAttributes.attribute2double(newRawValue.getRawValue());
-      } else if (param.getType() == String.class) {
-        value = HelperAttributes.attribute2string(newRawValue.getRawValue());
-      } else {
-        value = HelperAttributes.attribute2JavaType(newRawValue.getRawValue());
-      }
-      try {
-        param.set(this, value);
-      } catch (IllegalArgumentException | IllegalAccessException ex) {
-        Logger.getLogger(MonitorAndControlNMFAdapter.class.getName()).log(Level.SEVERE,
-            null, ex);
-        result = false;
-      }
+      result = result && onSetValue(newRawValue);
     }
     return result;
   }
+   
+  public Boolean onSetValue(ParameterRawValue newRawValue)
+  {
+    Object value;
+    Logger.getLogger(MonitorAndControlNMFAdapter.class.getName()).log(Level.INFO, "SET VALUE CALLED");
```

Comment:
Either remove this print altogether, or make it FINE level and more informative.